examples/Arangopipe_with_Pytorch_Example.ipynb

```diff
@@ -1,8 +1,10 @@
 {
```

Comment:
Are those changes intended here?

Reply:
@joerg84, no replaced with the version from master and repushed.